functions/senderDashboardDataIndexer/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "sender-dashboard-data-indexer-lambda",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "type": "commonjs",
+  "scripts": {
+    "test-build-old": "npm run-script test && npm run-script coverage && npm run-script sonar && npm run-script build",
```

Comment:
Can we remove this script, if it is not used anymore?

Reply:
This script is currently marked as old because it contains the Sonar script, which is not implemented at this time. However, it may be implemented in the future.